templates/intro-services/README.ipynb

```diff
@@ -0,0 +1,287 @@
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### Service UI Overview\n",
+    "\n",
+    "Navigate to your newly created Service in the Anyscale UI (`Home > Services > my_service`). It should be in \"Starting\" state. Click into it and wait for the service to enter \"Active\" state.\n",
+    "\n",
+    "You should see the service state, key metrics, and system event logs on the overview page.\n",
+    "\n",
+    "<img src=\"assets/service-overview.png\" height=400px>"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### Query from the public Internet\n",
+    "\n",
+    "Once the service is up, you can query the service from the public Internet using the same logic as when testing it locally, with two changes:\n",
+    "1. Update the `HOST` to the service endpoint.\n",
+    "2. Add the authorization token as a header in the HTTP request.\n",
+    "\n",
+    "Both of these values are printed when you run `serve publish`. You can also find them on the service page. For example, if the output looks like:\n",
+    "```bash\n",
+    "(anyscale +4.0s) You can query the service endpoint using the curl request below:\n",
+    "(anyscale +4.0s) curl -H 'Authorization: Bearer 26hTWi2kZwEz0Tdi1_CKRep4NLXbuuaSTDb3WMXK9DM' https://stable_diffusion_app-4rq8m.cld-ltw6mi8dxaebc3yf.s.anyscaleuserdata-staging.com\n",
+    "```\n",
+    "\n",
+    "Then:\n",
+    "- The authorization token is `26hTWi2kZwEz0Tdi1_CKRep4NLXbuuaSTDb3WMXK9DM`.\n",
+    "- The service endpoint is `https://stable_diffusion_app-4rq8m.cld-ltw6mi8dxaebc3yf.s.anyscaleuserdata-staging.com`."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import requests\n",
+    "\n",
+    "HOST = \"TODO_INSERT_YOUR_SERVICE_HOST\"\n",
+    "TOKEN = \"TODO_INSERT_YOUR_SERVICE_TOKEN\"\n",
+    "\n",
+    "def send_request(name: str) -> str:\n",
+    "    response: requests.Response = requests.get(\n",
+    "        f\"{HOST}/hello\",\n",
+    "        params={\"name\": name},\n",
+    "        headers={\n",
+    "            \"Authorization\": f\"Bearer {TOKEN}\",\n",
+    "        },\n",
+    "    )\n",
+    "    response.raise_for_status()\n",
+    "    return response.content"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "print(send_request(\"Theodore\"))"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Monitoring production services\n",
+    "\n",
+    "Along with the monitoring tools that come with Workspaces, in Services you also get a number of built-in metrics out of the box in the `Metrics` tab. This tab includes aggregated metrics across all rollouts for the service (possibly from multiple Ray clusters).\n",
```

Comment:
We should add a note here about using the serve logger as well (and in the code example).

Reply:
Let's have Gene add this once the logging UI is available?

Reply:
I can add an example.

@akshay-anyscale can you explain to me what I should note here? If I use `print()`, will that not show up in the logs at all?

Reply:
Ok maybe I'll just merge this for now to unblock things.

Reply:
it will show up but it won't be annotated with metadata like request id, etc. or follow any formatting (if we want to use json formatting). I agree we can merge and ask Gene to update this